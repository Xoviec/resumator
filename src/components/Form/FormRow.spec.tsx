import { render } from "@testing-library/react";
import { FormRow } from ".";

describe("FormRow", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<FormRow>test-content</FormRow>);

    expect(asFragment()).toMatchSnapshot();
  });
});
