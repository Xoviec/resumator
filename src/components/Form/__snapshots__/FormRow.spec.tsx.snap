// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormRow should render correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-5ax1kt"
  >
    test-content
  </div>
</DocumentFragment>
`;
